packages/solid/src/generators/library/lib/add-project.ts

```diff
@@ -0,0 +1,59 @@
+import { NormalizedSchema } from '../schema';
+import { ProjectType } from '@nrwl/workspace';
+import {
+  addProjectConfiguration,
+  getWorkspaceLayout,
+  TargetConfiguration,
+  Tree,
+} from '@nrwl/devkit';
+
+export function addProject(tree: Tree, options: NormalizedSchema) {
+  const targets: { [key: string]: TargetConfiguration } = {
+    lint: createLintTarget(options),
+  };
+
+  if (options.buildable || options.publishable) {
+    targets.build = createBuildTarget(tree, options);
+  }
+
+  addProjectConfiguration(tree, options.name, {
+    root: options.projectRoot,
+    sourceRoot: `${options.projectRoot}/src`,
+    projectType: ProjectType.Library,
+    tags: options.parsedTags,
+    targets,
+  });
+}
+
+function createBuildTarget(
+  tree: Tree,
+  options: NormalizedSchema
+): TargetConfiguration {
+  const { libsDir } = getWorkspaceLayout(tree);
+  return {
+    executor: '@nxext/solid:build',
```

Comment:
Should be @nxext/vite or you have to change the executors in package.json to the right filename. In package.json is: "builders": "./builders.json",
but you have a executors.json

Reply:
Ok, I changed `package.json` to:

```
  "schematics": "./generators.json",
  "executors": "./executors.json",
```